pallets/committee/src/lib.rs

```diff
@@ -430,6 +464,15 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
     /// Accepts or rejects the proposal if its threshold is satisfied.
     fn check_proposal_threshold(proposal: T::Hash) {
         if let Some(voting) = Self::voting(&proposal) {
+            // Make sure we don't have an expired proposal at this point.
+            if let Err(_) = Self::ensure_not_expired(
+                &proposal,
+                voting.expiry,
+                system::Module::<T>::block_number(),
+            ) {
```

Comment:
this can be re-written as 

```
if Self::ensure_not_expired(
        &proposal,
        voting.expiry,
        system::Module::<T>::block_number(),
      ).is_err() {
       return;
}
```

Reply:
Sure, but I don't think that makes anything clearer. Direct pattern matching both highlights the pattern as is more efficient to compile (no inlining necessary on LLVM's part).